src/test/java/io/harness/cf/client/api/dispatchers/Endpoints.java

```diff
@@ -2,7 +2,7 @@
 
 public class Endpoints {
 
-  static final String AUTH_ENDPOINT = "/api/1.0/client/auth";
+  public static final String AUTH_ENDPOINT = "/api/1.0/client/auth";
```

Comment:
Very very minor nit: any reason why this has been made explicit but the others haven't ?

Reply:
it was the only one i needed public since the test was in a different package - i've adjusted the others now

Reply:
Ah sorry - I thought not providing an access modifier defaulted to public. My mistake.